rules/test.bzl

```diff
@@ -80,6 +153,9 @@ def _ios_test(name, test_rule, test_suite_rule, apple_library, infoplists_by_bui
         **ios_test_kwargs
     )
 
+def split_ios_unit_test(name, split_name_to_kwargs, apple_library = apple_library, **kwargs):
```

Comment:
Could we merge it with ios_unit_test? We can check if split_name_to_kwargs exists to decide if it should call _ios_test or _split_ios_test.

Also could you update the document?

Reply:
Unless we can somehow mitigate this - the downside of changing it from what you have here is now you'll end up recompiling an entirely different `apple_library` per `slice` - where the existing code compiles a test 1 time: e.g. 1 `apple_library` per a set of `srcs`

Reply:
@jerrymarino  i don't think so if we still only compile a single apple_library before "splitting" them i would expect it would still act the same as it does today